dev/fix-path-variable-on-windows.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Because $OSTYPE could be cygwin, msys, or win32, etc on Windows. The possibilities are endless.
+if [[ "${OSTYPE}" != linux* && "${OSTYPE}" != darwin* ]]; then
+  export PATH=\
+"${PATH}:\
+$(
+  R_HOME='/c/R'
+  MINICONDA_ENVS_DIR='/c/Miniconda/envs'
+  MLFLOW_CONDA_ENV_NAME="$(${R_HOME}/bin/Rscript -e 'cat(mlflow:::mlflow_conda_env_name())')"
+  ADDITIONAL_PATHS_ARR=(
+    "${R_HOME}/bin"
+    "${MINICONDA_ENVS_DIR}"/{'test-environment',"${MLFLOW_CONDA_ENV_NAME}"}/'Scripts'
```

Comment:
What currently happens is required executables such as `mlflow.exe` and `waitress-serve.exe` are placed in `Miniconda/envs/Scripts` but the `Scripts` directory is not automatically part of `PATH`, so I had to create this bash script fixing `PATH` variable on Windows.

I think we need to make this as part of MLflow installation steps on Windows. What is the best way to document this type of info for MLflow Windows users?

Reply:
Actually, on my Windows 10 laptop `waitress-serve.exe` is placed in `Miniconda/envs/r-mlflow-1.17.0/Scripts`